src/analyzer/protocol/vxlan/VXLAN.cc

```diff
@@ -75,25 +75,16 @@ void VXLAN_Analyzer::DeliverPacket(int len, const u_char* data, bool orig,
 	Packet pkt(DLT_EN10MB, &ts, caplen, len, data);
 	pkt.encap = outer;
 
-	packet_mgr->ProcessPacket(&pkt);
-
-	if ( ! pkt.l2_valid )
+	if ( ! packet_mgr->ProcessInnerPacket(&pkt) )
```

Comment:
Can `bool l2_valid;` now be removed from `Packet.h`?

Reply:
Yep, removed.